install.sh

```diff
@@ -120,13 +120,12 @@ else
   if [ "$CONT" = "y" ]; then
     echo "Downloading example data from Dispatch repository..."
     curl -O https://raw.githubusercontent.com/Netflix/dispatch/latest/data/dispatch-sample-data.dump
-    export PGPASSWORD='dispatch'
     echo "Dropping database dispatch if it already exists..."
-    dropdb -h localhost -p 5432 -U dispatch dispatch --if-exists
+    docker-compose run -e PGPASSWORD='dispatch' --rm postgres dropdb -h postgres -p 5432 -U dispatch dispatch --if-exists
     echo "Creating dispatch database..."
-    createdb -h localhost -p 5432 -U dispatch dispatch
+    docker-compose run -e PGPASSWORD='dispatch' --rm postgres createdb -h postgres -p 5432 -U dispatch dispatch
     echo "Loading example data to the database..."
-    psql -h localhost -p 5432 -U dispatch -d dispatch -f ./dispatch-sample-data.dump
+    docker-compose run -e PGPASSWORD='dispatch' -v "$(pwd)/dispatch-sample-data.dump:/dispatch-sample-data.dump" --rm postgres psql -h postgres -p 5432 -U dispatch -d dispatch -f /dispatch-sample-data.dump
```

Comment:
We now have 4 instances of `dispatch-sample-data.dump` in the script. We store it in a variable and use the variable instead.

Reply:
Sure give me a sec. Gonna update my merge request.